docs/api_requests/patch-block.md

```diff
@@ -1,6 +1,6 @@
 # Updating a Block Device
 
-Attached block devices require a PATCH /drives API call when the backing
+Attached block devices require a `PATCH /drives/{id}` API call when the backing
```

Comment:
I would add a "docs: fix code format" in the commit title so that it points to a more specific destination in the project.

Reply:
Yes. I really should break this out to a separate patch.

Reply:
As far as my concern, you can do it in this PR.

Reply:
I may have not been clear enough. I was referring to the title in the commit not the title of the PR. You can change it by doing `git commit --amend` and then `git push -f`. You also need to do a rebase. Let me know if you need help.